### DISCUSSION TITLE
technology/works/node-cli

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#42] [Announcements]
# technology/works/node-cli

粥里有勺糖的个人博客，记录随笔与学习笔记，大前端相关的知识，高频面试题，个人面经等

http://localhost:4000/technology/works/node-cli.html

**Comment by ATQQ** [comment]
测试1

**Reply by ATQQ** [reply]
测试回复内容:smile:

**Comment by ATQQ** [comment]
测试2

**Comment by ATQQ** [comment]
更多回复

**Comment by ATQQ** [comment]
测试截图
